numbat/src/parser.rs

```diff
@@ -2,29 +2,56 @@
 //!
 //! Grammar:
 //! ```txt
-//! statement       →   expression | variable_decl | function_decl | dimension_decl | unit_decl | procedure_call | module_import
+//! statement       →   variable_decl | function_decl | dimension_decl | decorator | unit_decl | module_import | procedure_call | expression
```

Comment:
> `decorator`

I think this is wrong? A standalone decorator is not a statement. It can precede a unit definition. And together with the unit definition, it forms a statement.